content/millennium/r4/individuals/related-person.md

```diff
@@ -26,7 +26,7 @@ The following fields are returned if valued:
 * [Date of birth](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.birthDate){:target="_blank"}
 * [Address](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.address){:target="_blank"}
 * [Communication (preferred language)](https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.communication){:target="_blank"}
-* [Extensions including related person encounter and relationship level](#extensions){:target="_blank"}
+* [Extensions including related person encounter,relationship level and period](#extensions){:target="_blank"}
```

Comment:
nit: missing a space after the first comma

Reply:
added space here https://github.com/cerner/fhir.cerner.com/pull/624/commits/1be40a0943e52ebd719a02f2cfeefcdbadac0c82#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR29